### PR TITLE
Improve Readme file: add require at application.rb file

### PR DESCRIPTION
**Pull request by TiSer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ This gem provides methods for generating container tags, such as `div`, for your
 
 Simply add `gem 'record_tag_helper', '~> 1.0'` to your Gemfile and run `bundle install`.
 
+You may need to add `require "record_tag_helper"` to your `application.rb` file.
```
Comment (rafaelfranca) [inline README.md:13]: Why do you need to do that? `gem 'record_tag_helper'` in the Gemfile will be automatically required by a Rails application.

Comment (TiSer) [reply README.md:13]: I had this issue in my application, that wasn't required automatically and I spent much time investigating why it doesn't work.

Comment (rafaelfranca) [reply README.md:13]: So maybe you have a bug in your application? The current writing of the README is correct for most Rails applications and I don't want to add something to the readme that is caused by a bug in the application.

Comment (TiSer) [reply README.md:13]: Ok, excuse me for interrupting if so.

Comment (rafaelfranca) [reply README.md:13]: It is fine. I not saying you have a bug, I'm saying maybe you have a bug. Can you check why in your application you need to require? You could for example try to create a new app and include this gem to see if it works or you also have to require.

**Pull request closed without merging**